core/storage/suggestion/gae_models.py

```diff
@@ -229,6 +235,27 @@ def query_suggestions(cls, query_fields_and_values):
 
         return query.fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_translation_suggestions_with_exp_ids(cls, exp_ids):
```

Comment:
Just to check, have you exercised this function manually (via manual testing on your local devserver)? If so, could you please describe how you verified that it was actually called?

Reply:
OK, please test manually (with a local dev server) and confirm that you've done so. Do this in the future for all PRs.

Reply:
Hi Sean - do you mean manually test any new methods that I implement in storage models specifically?  To test that ` get_translation_suggestions_with_exp_ids' was working properly, I put an exception in `auto_reject_translations_with_exp_target_ids` after `get_translation_suggestions_with_exp_ids(exp_ids)` to check that the translated content of the suggestions matched what I entered. I logged on to the devserver as an admin, loaded some explorations that were associated with a topic, submitted some translation suggestions associated with that topic and then deleted that topic. The translations that I entered matched the thrown exception.
Here are some screenshots:
<img width="1447" alt="Screen Shot 2020-08-03 at 3 51 18 PM" src="https://user-images.githubusercontent.com/19376937/89236988-f03c6a00-d5a6-11ea-88b3-bba5642cf141.png">
<img width="1986" alt="Screen Shot 2020-08-03 at 3 51 00 PM" src="https://user-images.githubusercontent.com/19376937/89236992-f29ec400-d5a6-11ea-843e-77f55d7b6a1c.png">

Reply:
Great -- yes, that's what I meant. After you did this, did you see any changes to index.yaml when running `git status`? If so, please commit them too.

See the first paragraph [here](https://cloud.google.com/appengine/docs/standard/python/config/indexref) for more background.

Reply:
I didn't see any changes in the index.yaml file no

Reply:
OK, thanks for checking! I am a bit surprised by that, but if you don't see changes to that file when testing this function on a local dev server, then I guess that's fine.

(Rationale for the original concern: there's no target_id field in index.yaml, so since this is querying on multiple fields, I would have expected that a composite index would need to be generated.)